README.md

```diff
@@ -9,39 +9,41 @@ See https://georchestra.geo-solutions.it/ (work in progress)
 
 ## Building and install the project
 
-### Prerequisite 
+### Prerequisite
 
-**1. Clone the repository**
+**1. Requirements**
 
-`git clone --recursive https://github.com/georchestra/mapstore2-georchestra.git`
+You need to read the relevant requirements documentation according to your installation :
+
+https://docs.mapstore.geosolutionsgroup.com/en/latest/developer-guide/requirements/
+
+To read the appropriate requirements page, open the previous link and select the corresponding MapStore2 release in the bottom right corner.
 
-**2. Install NodeJS**
+**2. Clone the repository**
 
-* Suggested NodeJS 12.x, npm 6.x.
-* Minimal version : NodeJS >= 10, npm >= 6. 
-* NodeJS > 14.x is not currently supported
+`git clone --recursive https://github.com/georchestra/mapstore2-georchestra.git`
+
+**3. Install NodeJS**
 
 :warning: For minimal you may need to increase used memory limits.
 
 To get more informations about nodejs install :
 
 * Instructions to install Nodejs from package manager : [here](https://nodejs.org/en/download/package-manager/)
 * Download source : [here](https://nodejs.org/en/download/)
-* Download a specific version : [here](https://nodejs.org/dist/)
-
-**3. Install Java SDK**
+* Download a specific version : [here](https://nodejs.org/dist/) (v16 is recommended)
```

Comment:
well.. maybe *required* instead of *recommended* then ?

Reply:
Mapstore doc just "recommend" (i guess this works with v14).

![image](https://github.com/georchestra/mapstore2-georchestra/assets/16317988/7b57ca88-ed6c-41d5-b826-4586edf5de28)

I can change to "required" instead.

Reply:
given that minimum is 16 above, and https://github.com/georchestra/mapstore2-georchestra/issues/672#issuecomment-1831729359 id be inclined to require 16 :)